.travis.yml

```diff
@@ -14,6 +14,8 @@ matrix:
           env: TOXENV=py36
```

Comment:
I missed this on my first pass. [Python 3.4 is now EOLed](https://www.python.org/dev/peps/pep-0429/#release-schedule). Suggest its removal as well, both here and below.